### ISSUE TITLE
Transfer learning tutorial -- momentum handling

### ISSUE THREAD

**Issue by lizhitwo** [pytorch/tutorials#61]
In the tutorial, a new SGD optimizer is generated each epoch, which means that the momentum from the last epoch is discarded. As far as I can tell, this is unusual behavior in deep network training. The same optimizer should be used, but with learning rate modified.

**Comment by chsasank** [comment]
There's a discussion about this on pytorch forums [here](https://discuss.pytorch.org/t/adaptive-learning-rate/320/8)

@apaszke suggests 
> If you want to change the LR we recommend reconstructing the optimizer with new parameters.

Your issue has also been raised in that thread, although there has been no conclusive answer yet. May be @apaszke or @soumith will help us with this?

**Comment by lizhitwo** [comment]
Thanks! It seems the best way for reproducing the usual behavior right now is to manually adjust the learning rate for each group.

**Comment by chsasank** [comment]
Hi @lizhitwo,

I've created a PR #62 for this. Can you please have a look before I merge and build?

**Comment by lizhitwo** [comment]
This looks right. Thanks!

**Comment by chsasank** [comment]
Thanks for raising the issue.
